lib/compress/zstd_compress_internal.h

```diff
@@ -170,6 +170,7 @@ struct ZSTD_CCtx_params_s {
 struct ZSTD_CCtx_s {
     ZSTD_compressionStage_e stage;
     int cParamsChanged;                  /* == 1 if cParams(except wlog) or compression level are changed in requestedParams. Triggers transmission of new params to ZSTDMT (if available) then reset to 0. */
+    int bmi2;
```

Comment:
Could you add a comment which explains how this variable is initialized and used ?
It's perfectly visible in this patch, because all relevant parts are grouped together, but outside of this PR, it might be harder to follow.